tests/jballerina-unit-test/src/test/resources/test-src/runtime/api/async/main.bal

```diff
@@ -39,6 +39,10 @@ public class ObjectMethodsCallClass {
         }
     }
 
+    function getFieldValWithMultipleOptionalArgs(int a = 3, int b = a, int c = a + b) returns int {
```

Comment:
Shall we add some tests with method calls (both async and non-async calls) for default values?

Reply:
Added asunc call tests.